fusor-ember-cli/app/controllers/rhev-options.js

```diff
@@ -42,30 +42,34 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
        }
   ],
 
-  invalidIsAlphaNumericRhevDatabase: Ember.computed('rhev_database_name', function() {
+  invalidIsAlphaNumericRhevDatabase: Ember.computed('rhevDatabaseName', function() {
       var rx = new RegExp(/^[A-Za-z0-9_-]+$/);
-      if (Ember.isPresent(this.get('rhev_database_name'))) {
-          return !(this.get('rhev_database_name').match(rx));
+      if (Ember.isPresent(this.get('rhevDatabaseName'))) {
+          return !(this.get('rhevDatabaseName').match(rx));
       }
   }),
 
-  invalidIsAlphaNumericRhevCluster: Ember.computed('rhev_cluster_name', function() {
+  invalidIsAlphaNumericRhevCluster: Ember.computed('rhevClusterName', function() {
       var rx = new RegExp(/^[A-Za-z0-9_-]+$/);
-      if (Ember.isPresent(this.get('rhev_cluster_name'))) {
-          return !(this.get('rhev_cluster_name').match(rx));
+      if (Ember.isPresent(this.get('rhevClusterName'))) {
+          return !(this.get('rhevClusterName').match(rx));
       }
   }),
 
   disableNextRhevOptions: Ember.computed(
-    'rhev_root_password',
-    'rhev_engine_admin_password',
+    'rhevRootPassword',
+    'confirmRhevRootPassword',
+    'confirmRhevEngineAdminPassword',
+    'confirm_rhev_engine_admin_password',
```

Comment:
@cfchase, I think you need to make this change. If I change the rhev admin password to not match (after it matched), the Next button does not disable as it should.

```
+    'rhevEngineAdminPassword',
-    'confirm_rhev_engine_admin_password',
````